docs/sources/panels-visualizations/visualizations/xy-chart/index.md

```diff
@@ -0,0 +1,264 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/xy-chart/
+keywords:
+  - grafana
+  - chart
+  - xy chart
+  - documentation
+  - guide
+  - graph
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+description: Configure options for Grafana's xy chart
+title: XY chart
+weight: 100
+---
+
+# XY chart
+
+XY charts provide a way to visualize arbitrary x and y values in a graph so that you can easily show the relationship between two variables.
```

Comment:
```suggestion
XY charts provide a way to visualize arbitrary x and y values in a graph so that you can easily show the relationship between two variables. XY charts are typically used to create scatter plots. You can also use them to create bubble charts where size fields are mapped to the points or bubbles in the visualization. 
```